tests/test_specs.py

```diff
@@ -17,67 +17,67 @@
 
 
 def test_one_point_line() -> None:
-    x = object()
+    x = "x"
```

Comment:
Can we move this out as a global for all tests please?

```python
x = "x"

def test_something() -> None:
    inst = Line(x, 0, 1, 2)
    ...

def test_something_else() -> None:
    inst = Line(x, 2, 5, 2)
    ...
```

Reply:
updated